src/components/molecules/Card-Product/CardProduct.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Icon from '../../atoms/Icon/Icon';
+import Button from '../../atoms/Button/Button';
+import './CardProduct.scss';
+import {useTranslation} from 'react-i18next';
+import { GlobalContext } from '../../../Context/globalContext';
+const CardProduct = ({details}) =>{
+    const {dispatch} = React.useContext(GlobalContext);
+    const {t} = useTranslation();
+    const addToCart = () =>{        
+        dispatch({
+            type:'ADD_ITEM',
+            payload:{...details, quantity:1, totalPrice:details.price}
+        });
+    }
+    return (
+        <>
+    <div className="product" tabIndex="0" role="card">
```

Comment:
Kindly share the thought. are you trying to make a card focusable element?

Reply:
Yes Salil, complete product card on focus should also be detectable supposedly. We can surely discuss upon it.